contracts/0.8.9/WithdrawalQueue.sol

```diff
@@ -57,17 +57,14 @@ contract WithdrawalQueue is AccessControlEnumerable, WithdrawalQueueBase {
     using UnstructuredStorage for bytes32;
 
     ///! STRUCTURED STORAGE OF THE CONTRACT
-    ///  Inherited from AccessControlEnumerable:
-    ///! SLOT 0: mapping(bytes32 => RoleData) _roles
-    ///! SLOT 1: mapping(bytes32 => EnumerableSet.AddressSet) _roleMembers
     ///  Inherited from WithdrawalQueueBase:
-    ///! SLOT 2: mapping(uint256 => WithdrawalRequest) queue
-    ///! SLOT 3: uint256 lastRequestId
-    ///! SLOT 4: uint256 lastFinalizedRequestId
-    ///! SLOT 5: mapping(uint256 => Discount) discountHistory
-    ///! SLOT 6: uint256 lastDiscountIndex
-    ///! SLOT 7: uint128 public lockedEtherAmount
-    ///! SLOT 8: mapping(address => uint256[]) requestsByRecipient
+    ///! SLOT 0: mapping(uint256 => WithdrawalRequest) queue
```

Comment:
:+1: 
The endgame is to have unstructured contract I suppose (like `StakingRouter`)

Reply:
Yep. But I will do it in another PR later.